### PR TITLE
Fix warnings

### PR DESCRIPTION
**Pull request by TrySound**
We can simplify it just checking `result.warnings()[0].text` without plugin prefix

### CODE REVIEW

File: test/index.js [hunk 1]
```diff
@@ -66,7 +70,12 @@ test('absolute', function(t) {
 	};
 	clearResults('test/result/copy.css', 'test/imported');
 	return compareFixtures(t, 'should change existing assets path', rebaserOptions, postcssOptions)
-		.then(function () {
+		.then(function (result) {
```
Comment (cakeinpanic) [inline test/index.js]: i don't like this repeating part of code with

```
t.equal(result.warnings()[0].toString(), [
```

I suggest to move this check to standalone method

**Comment by TrySound** [comment]
@cakeinpanic  Squash?

**Comment by TrySound** [comment]
Is it okay now?

**Comment by TrySound** [comment]
@cakeinpanic Any thoughts?

**Comment by cakeinpanic** [comment]
Sorry for waiting so long(
Thank you!

**Pull request merged**